samples/bluetooth/nrf_dm/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -7,3 +7,15 @@
 &ipc0 {
 	zephyr,buffer-size = <1536>;
 };
+
+ /* Enable pin forwarding to network core. The selected pins will be used by
+  * Distance Measurement module for debug purposes.
+  *
+  * Note: Pay attention to assign the same GPIO pins as Kconfig file.
+  */
+&gpio_fwd {
+	dfe-gpio-if {
+		gpios = <&gpio0 26 0>,
```

Comment:
Should we add a comment saying that this needs to be equal to DM_RANGING_PIN and DM_ADD_REQUEST_PIN?
Additionally, would it be possible to somehow solve this in the subsys, since the config comes from there?